exonum/src/proto/schema/exonum/runtime.proto

```diff
@@ -44,10 +44,17 @@ message InstanceSpec {
   ArtifactId artifact = 3;
 }
 
+enum ErrorKind {
```

Comment:
Add description please.

Reply:
It would be helpful as it will be inserted in the generated classes which may be used in user code: ECR-3731